modules/minimint.nix

```diff
@@ -0,0 +1,125 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  options.services.minimint = {
+      enable = mkOption {
+      type = types.bool;
+      default = true;
+      description = ''
+        Enable Minimint,is a federated Chaumian e-cash mint backed 
+        by bitcoin with deposits and withdrawals that can occur on-chain
+        or via Lightning.
+      '';
+    }; 
+    address = mkOption {
+      type = types.str;
+      default = "127.0.0.1";
+      description = "Address to listen for RPC connections.";
+    };
+    port = mkOption {
+      type = types.port;
+      default = 5001;
+      description = "Port to listen for RPC connections.";
+    };
+    extraArgs = mkOption {
+      type = types.separatedString " ";
+      default = "";
+      description = "Extra command line arguments passed to electrs.";
+    };
+    dataDir = mkOption {
+      type = types.path;
+      default = "/var/lib/minimint";
+      description = "The data directory for minimint.";
+    };
+    user = mkOption {
+      type = types.str;
+      default = "minimint";
+      description = "The user as which to run minimint.";
+    };
+    group = mkOption {
+      type = types.str;
+      default = cfg.user;
+      description = "The group as which to run minimint.";
+    };
+    tor.enforce = nbLib.tor.enforce;
```

Comment:
I think we agreed on not enforcing Tor connections between federation members, what does this do?